tests/AbstractSnowflakeTest.php

```diff
@@ -155,6 +155,8 @@ private function setupTables(): void
                       OBJECT_CONSTRUCT(\'a\', 1, \'b\', \'BBBB\', \'c\', null) AS "org",
                       ARRAY_CONSTRUCT(10, 20, 30) AS "arr";'
         );
+
+        $this->connection->query("DROP TABLE IF EXISTS \"PUBLIC\".\"no_schema_escaping\"");
```

Comment:
This is never created, so I think it's enough that it has been ran at least once with this query and now can be safely removed.